runner/run.go

```diff
@@ -230,10 +233,19 @@ func markAndFlush(runContext *TestRunContext, dest *ftwhttp.Destination, stageID
 		Version: "HTTP/1.1",
 	}
 
+	// check if destination host is an alias for localhost
+	addrs, err := net.LookupHost(dest.DestAddr)
+	if err != nil {
+		return nil, fmt.Errorf("ftw/run: can't resolve destination %+v: %w", dest, err)
+	}
```

Comment:
Don't know if this even make sense. Maybe just ignoring the error and moving on?